src/RA_Leaderboard.cpp

```diff
@@ -176,17 +168,7 @@ void RA_Leaderboard::SubmitRankInfo(unsigned int nRank, const std::string& sUser
 
 void RA_Leaderboard::SortRankInfo()
 {
-    for (size_t i = 0; i < m_RankInfo.size(); ++i)
-    {
-        for (size_t j = i + 1; j < m_RankInfo.size(); ++j)
-        {
-            if (m_RankInfo.at(i).m_nRank > m_RankInfo.at(j).m_nRank)
-            {
-                Entry temp = m_RankInfo[i];
-                m_RankInfo[i] = m_RankInfo[j];
-                m_RankInfo[j] = temp;
-            }
-        }
-    }
+    static_assert(ra::is_nothrow_lessthan_comparable_v<decltype(m_RankInfo)>);
+    std::sort(m_RankInfo.begin(), m_RankInfo.end());
```

Comment:
`N^2 logN` to `N logN`. The static_assert should be self-explanatory, but the test asserts at compile-time that `RA_Leaderboard::Entry` is less than comparable. Meaning it can be compared with the less than operator (`operator<`)
Since std::sort uses `std::less()(value_type, value_type)`, which is a function object of `operator<`.